### DISCUSSION TITLE
How to use in Angular?

### DISCUSSION THREAD

**Discussion by laynghiem** [storybookjs/addon-designs#105] [Q&A] [answered]
Hi,

I succesfully installed this add-on, but I don't know how to include a story with a figma link in
typescript. Do you guys have any examples?

Kind regards.

**Comment by pocka** [comment, marked as answer]
[Here is a link for the quick start page](https://pocka.github.io/storybook-addon-designs/?path=/story/docs-quick-start--page).

All you need to do is attaching `withDesign` [decorator](https://storybook.js.org/docs/angular/writing-stories/decorators) to your stories and setting `design` [parameter](https://storybook.js.org/docs/angular/writing-stories/parameters). Although all the demos are written with `@storybook/react`, both the Parameter and the Decorator is a framework-agnostic concept of Storybook so you can use it in the same manner in other frameworks.

**Reply by laynghiem** [reply]
Thank you 😊
